proto/api/v0_2/sink/sink_service.proto

```diff
@@ -0,0 +1,26 @@
+syntax="proto3";
+
+package yandexcloud.datatransfer.dtextension.v0_2.sink;
+
+import "api/v0_2/common.proto";
+import "api/v0_2/sink/control.proto";
+
+service SourceService {
+    // Spec is method for determining endpoint settings available for configuring
+    rpc Spec(SpecReq) returns (SpecRsp) {}
+    // Check is method that checks settings of endpoint are comply to specification
+    // In order to see specification see Spec handle
+    rpc Check(CheckReq) returns (CheckRsp) {}
+
+    rpc Write(stream WriteReq) returns (stream WriteRsp) {}
```

Comment:
SourceService should be readonly

Reply:
You mean SinkService? It can be readonly indeed